### PR TITLE
Fix IpfixCollectorCfg reload, fix some compiler warnings, add more debugs, clarify other debug messages

### PR DESCRIPTION
**Pull request by bluca**
See commit messages for more details

### CODE REVIEW

File: src/modules/ConfigManager.cpp [hunk 1]
```diff
@@ -310,12 +310,12 @@ Graph* ConfigManager::reconnect(Graph* g, Graph *old)
...
-                    delme.delete_after = time(NULL) + DELETER_DELAY; // current time + 20 seconds
-                    deleter_list.push_back(delme);
-                    msg(MSG_INFO, "can't reuse %s(id=%u)",
-							oldCfg->getName().c_str(), oldCfg->getID());
+					deleter_list_item delme;
+					delme.c = oldCfg;
+					delme.delete_after = time(NULL) + DELETER_DELAY; // current time + 20 seconds
+					deleter_list.push_back(delme);
+					msg(MSG_INFO, "can't reuse %s(id=%u)",
+					    oldCfg->getName().c_str(), oldCfg->getID());
```
Comment (ogasser) [inline src/modules/ConfigManager.cpp]: Mix of tabs and spaces -> wrong indentation.

File: src/modules/ipfix/CollectorCfg.h [hunk 2]
```diff
@@ -150,6 +150,8 @@ class CollectorCfg
 			THROWEXCEPTION("Could not create IpfixReceiver");
 		}
 
+		msg(MSG_INFO, "Created an IpfixReceiver: %d", protocol);
```
Comment (ogasser) [inline src/modules/ipfix/CollectorCfg.h]: Better: `"Created an IpfixReceiver for protocol %d"`

File: src/modules/ConfigManager.cpp (hunk 1 above)
Comment (bluca) [inline src/modules/ConfigManager.cpp]: fixed

File: src/modules/ipfix/CollectorCfg.h (hunk 2 above)
Comment (bluca) [inline src/modules/ipfix/CollectorCfg.h]: Fixed

**Comment by ogasser** [comment]
Great! :+1:

**Comment by bluca** [comment]
Thanks! One more change to help with debugging receivers: https://github.com/tumi8/vermont/pull/62

**Pull request merged**